src/TortoiseProc/GitLogListBase.cpp

```diff
@@ -95,8 +95,16 @@ CGitLogListBase::CGitLogListBase():CHintListCtrl()
 	m_bFilterWithRegex = !!CRegDWORD(_T("Software\\TortoiseGit\\UseRegexFilter"), FALSE);
 	m_bFilterCaseSensitively = !!CRegDWORD(_T("Software\\TortoiseGit\\FilterCaseSensitively"), FALSE);
 
-	m_From = -1;
-	m_To = -1;
+	m_Filter.m_NumberOfLogsScale = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogsScale"), -1);
+	m_Filter.m_NumberOfLogs = (DWORD)CRegDWORD(_T("Software\\TortoiseGit\\LogDialog\\NumberOfLogs"), 0);

```

Comment:
if those values are dropped, the "from" context menu will have the same value on commits/years/months/weeks.
It leads some context menu items are also needed to be dropped, and leave one item there.
maybe "Last 1000 commits", or "Last 1 year". (I accept this. :) )